cg/store/models.py

```diff
@@ -971,3 +972,42 @@ class Order(Base):
 
     def to_dict(self):
         return to_dict(model_instance=self)
+
+
+class RunDevice(Base):
+    """Model for storing run devices."""
+
+    __tablename__ = "run_device"
+
+    id: Mapped[PrimaryKeyInt]
+    type: Mapped[DeviceType]
+
+    __mapper_args__ = {
+        "polymorphic_on": "type",
```

Comment:
do we need a `polimorphic_identity` field here for the parent classes?

Reply:
Not strictly necessary unless we would like to call these tables directly according to what the docs say. Not sure how it will behave exactly. We can see and test.